src/main/java/controller/UsuarioController.java

```diff
@@ -3,36 +3,68 @@
 
 import business.UsuarioService;
 import data.entities.Usuario;
+import data.repositories.UsuarioRepository;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Do not use * imports.

Reply:
El IDE sugiere utilizar **import all** debido a que estoy utilizando **todas** las anotaciones de `import org.springframework.web.bind.annotation` .

Reply:
OK.

Reply:
Ok